### PR TITLE
Add no_doubles, data_partner and data_opponent hands

### PR DESCRIPTION
**Pull request by AntiD2ta**

### CODE REVIEW

File: domaino/domino/hands/data_oponent.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    Share a high data or a low data with a oponent
+    Randomly distribute pieces among other player.
+    Valid pieces are all integer tuples of the form:
+        (i, j) 0 <= i <= j <= max_number
+    Each player will have `pieces_per_player`.
+    """
+    if high:
+        data_num = randint(max_number // 2 + 1, max_number)
+    else:
+        data_num = randint(0, max_number // 2)
```
Comment (e1Ru1o) [inline domaino/domino/hands/data_oponent.py]:
```suggestion
    data_num = randint([0, max_number // 2 + 1][high], [max_number // 2, max_number][high])
```

File: domaino/domino/hands/data_oponent.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    data = [(min(data_num, i), max(data_num, i)) for i in sample(list(range(pieces_per_player)), randint(max_number - 1, max_number + 1) )]
```
Comment (e1Ru1o) [inline domaino/domino/hands/data_oponent.py]:
```suggestion
    data = [(min(data_num, i), max(data_num, i)) for i in sample(list(range(max_number + 1)), min(randint(max_number - 1, max_number + 1), pieces_per_player)]
```

File: domaino/domino/hands/data_oponent.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    assert 4 * pieces_per_player <= len(piecesO) + len(handO) + len(hand0)
+    hand = sample(piecesO, 2 * pieces_per_player)
+    temp_hands = [hand[i:i+pieces_per_player] for i in range(0, 2 * pieces_per_player, pieces_per_player)]
+    oponent_hands = [handO, temp_hands[0]]
+    shuffle(oponent_hands)
+
+    hands = [hand0]
+    hands += [oponent_hands[0]]
+    hands += [temp_hands[1]]
+    hands += [oponent_hands[1]]
```
Comment (e1Ru1o) [inline domaino/domino/hands/data_oponent.py]:
```suggestion
    hands = [hand0, oponent_hands[0], temp_hands[1], oponent_hands[1]]
```

File: domaino/domino/hands/data_oponent.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+    for h in hands:
+        print(h)
```
Comment (e1Ru1o) [inline domaino/domino/hands/data_oponent.py]:
```suggestion
```

**Review by e1Ru1o — CHANGES_REQUESTED** [review]
Please merge the behavior of data_oponent and data_partner in a util method

**Review by e1Ru1o — APPROVED** [review]

**Review by cicr99 — APPROVED** [review]

**Pull request merged**